### DISCUSSION TITLE
Inventory Data from Quote in List View Mode

### DISCUSSION THREAD

**Discussion by JEEVAN28** [YetiForceCompany/YetiForceCRM#15506] [How to]
Hi,

Is there any way to show List View with Inventory Lines. Below is scenario.
I have created 3 quotes in which 8 different / same Items are selected. now I want List View showing 8 lines with each inventory item and its quote details.

Regards
Jeevan Pendhari

**Comment by rskrzypczak** [comment]
The only place where you can view the block with products on the list of records is from the related record tab.
![image](https://user-images.githubusercontent.com/10221999/144368380-a6bc5638-927e-481c-859d-c6d879baec8c.png)
.. and you can manage the fields from the panel.
![image](https://user-images.githubusercontent.com/10221999/144368672-1b003255-11a3-4ac1-b9bf-0eca3bf07dcd.png)

There are no other views between the list view and the inventory items list in either direction of the relationship.

**Reply by JEEVAN28** [reply]
Ok, but this will not be available for analysis. Actually this is reporting part.
